casper/src/main/resources/Registry.rho

```diff
@@ -1,15 +1,26 @@
-new _registryStore,
-    lookupCh,
-    bootstrapLookup(`rho:registry:lookup`),
-    insertArbitraryCh,
-    bootstrapInsertArbitrary(`rho:registry:insertArbitrary`),
-    insertSignedCh,
-    bootstrapInsertSigned(`rho:registry:insertSigned:secp256k1`),
-    buildUri,
-    ops(`rho:registry:ops`),
-    TreeHashMap
+new
+  _registryStore,
+  lookupCh,
+  bootstrapLookup(`rho:registry:lookup`),
+  insertArbitraryCh,
+  bootstrapInsertArbitrary(`rho:registry:insertArbitrary`),
+  insertSignedCh,
+  bootstrapInsertSigned(`rho:registry:insertSigned:secp256k1`),
+  insertBlessed,
+  blessedContractLocMapCh,
+  buildUri,
+  ops(`rho:registry:ops`),
+  secpVerify(`rho:crypto:secp256k1Verify`),
+  blake2b256(`rho:crypto:blake2b256Hash`),
+  TreeHashMap,
+  TreeHashMapLoc,
+  TreeHashMapDispatcher,
+  secpVerify(`rho:crypto:secp256k1Verify`),
+  blake2b256(`rho:crypto:blake2b256Hash`),
+  hashVerify,
+  storeToken,
+  normalInsertSigned
```

Comment:
what does `normalInsertSigned` mean? I mean why it is `normal`?

Reply:
How would you like to call that?

Reply:
Why not just `insertSigned?`